includes/definitions/discovery/vertiv-pdu.yaml

```diff
@@ -0,0 +1,16 @@
+os: vertiv-pdu
```

Comment:
Hello
This file is not the correct one. You seems to have uploaded the definition twice, instead of pushing the discovery one here.

Reply:
I apologize, you are correct.  I will correct this.